python_modules/dagster/dagster/_core/execution/api.py

```diff
@@ -669,6 +680,7 @@ def _get_execution_plan_from_run(
         and execution_plan_snapshot.can_reconstruct_plan
```

Comment:
side note, not sure when this would be false. If the job was updated between us launching and executing the run?